lib/Data/Text/Slugger.hs

```diff
@@ -55,6 +55,19 @@ adjustChar 'Ł' = "l"
 adjustChar 'ß' = "ss"
 adjustChar 'þ' = "th"
 adjustChar 'Þ' = "th"
+adjustChar 'ç' = "c"
+adjustChar 'Ç' = "c"
+adjustChar 'ğ' = "g"
+adjustChar 'Ğ' = "g"
+adjustChar 'ı' = "i"
+-- See Note [Turkish I]
+adjustChar 'İ' = "i"
+adjustChar 'ö' = "o"
+adjustChar 'Ö' = "o"
+adjustChar 'ş' = "s"
+adjustChar 'Ş' = "s"
+adjustChar 'ü' = "u"
+adjustChar 'Ü' = "u"
```

Comment:
I would be surprised if a number of these aren't already handled by the ICU library and then us cleaning out the diacritic properties. For example, the French test covers `GARÇON - déjà , Forêt — Zoë` without having to adjust any special characters here.

Could you try commenting out all of these except the `Turkish I` and try running the tests again and see where they fail?

Reply:
If you don't have time, I can give it a go tonight!

Reply:
I gave it a go, and you are right! I missed the part where you remove the diacritics. As you mentioned, only `'ı' -> 'i'` seems to be necessary otherwise the letter is lost.

Reply:
Nice job!